vim/core/oni-plugin-git/index.js

```diff
@@ -31,8 +30,16 @@ const activate = Oni => {
         const isDir = await pathIsDir(filePath);
         const dir = isDir ? filePath : path.dirname(filePath);
         let branchName;
+        let summary;
         try {
           branchName = await Oni.services.git.getBranch(dir);
+          try {
+            summary = await Oni.services.git.getGitSummary(
+              Oni.workspace.activeWorkspace
+            );
+          } catch (e) {
+            console.warn('[Oni.Git.Plugin]: Could not get Summary', e);
```

Comment:
Is there a case where we could not the summary, but we could get the branch? If not, it might make sense to simplify and consolidate these try/catches

Reply:
That's essentially there for development purposes I'll remove that once the PR is finalised